readme.md

```diff
@@ -1,4 +1,26 @@
 ## 🛴 escooters
+### Available providers
+
+1. Beam
+1. Beryl
+1. BinBin
+1. Bird
+1. BitMobility
+1. Bolt
+1. Dott
+1. Hulaj
+1. Lime
+1. Link
+1. Neuron
+1. Quick
+1. Ryde
+1. Spin
+1. Tier
+1. Urent
+1. Veo
+1. Voi
+1. Zwings
```

Comment:
```suggestion
1. Beam
2. Beryl
3. BinBin
4. Bird
5. BitMobility
6. Bolt
7. Dott
8. Hulaj
9. Lime
10. Link
11. Neuron
12. Quick
13. Ryde
14. Spin
15. Tier
16. Urent
17. Veo
18. Voi
19. Zwings
```

Reply:
This was suggested in https://github.com/blumilksoftware/escooters/pull/117#discussion_r1299739789